### PR TITLE
Update save

### PR DESCRIPTION
**Pull request by huyqbui**
Chloe and I created save test buttons for each of the test menus (except security testing). 

The Save Test button on click will: 
  - Open export file modal if user has not save the test file yet
  - Overwrite and save the current test file on successive clicks

Added styling to match the neighboring button components for uniformity.

### CODE REVIEW

File: public/electron.jsx [hunk 1]
```diff
@@ -167,7 +171,7 @@ let githubWindow;
 // ipcMain is listening on channel 'Github-Oauth' for an event from ProjectLoader line 94
 // ipbMain receives the url from ProjectLoader.jsx line 94
 ipcMain.on('Github-Oauth', (_event, url) => {
-  console.log('opening github oauth window!!');
+  console.log('opening github oauth window');
```
Comment (alsdk850) [inline public/electron.jsx:174]: console.log is here!

File: src/pages/TestFile/TestFile.jsx [hunk 2]
```diff
@@ -109,6 +121,20 @@ const TestFile = () => {
     closeTestModal();
   };
 
+  // const saveTest = () => {
```
Comment (alsdk850) [inline src/pages/TestFile/TestFile.jsx:124]: maybe delete this whole comment block if not used?🤔

File: src/pages/TestFile/TestFile.jsx [hunk 3]
```diff
@@ -259,6 +286,14 @@ const TestFile = () => {
           </div>
         </Fragment>
       )}
+      {/* <br></br>
```
Comment (alsdk850) [inline src/pages/TestFile/TestFile.jsx:289]: Same with this one maybe we could delete this chunk of comment if it isn't necessary?!

**Review by alsdk850 — CHANGES_REQUESTED** [review]
Great job Chloe and Huy! 😄

File: public/electron.jsx (hunk 1 above)
Comment (huyqbui) [reply public/electron.jsx]: nice catch! removing it now

File: src/pages/TestFile/TestFile.jsx (hunk 2 above)
Comment (huyqbui) [reply src/pages/TestFile/TestFile.jsx]: forgot about these, removing them now

File: src/pages/TestFile/TestFile.jsx (hunk 3 above)
Comment (huyqbui) [reply src/pages/TestFile/TestFile.jsx]: got it, removing it now!

**Review by alsdk850 — APPROVED** [review]

**Review by Ruzeb — APPROVED** [review]
LGTM

**Pull request merged**